### PR TITLE
doc: update to CAF sensor manager

### PR DESCRIPTION
**Pull request by divipillai**
Added Single core configuration
section

### CODE REVIEW

File: samples/caf_sensor_manager/README.rst [hunk 1]
```diff
@@ -31,6 +32,24 @@ This module handles the data received from the :ref:`caf_sensor_data_aggregator`
 If you are running this sample on an SoC with multiple cores, the workload simulator module (``workload_sim``) is placed on the second core.
 All communication between the cores is done using :ref:`event_manager_proxy` and Zephyr subsystem :file:`include/ipc/ipc_service.h`.
 
+Configuration
+*************
+
+|config|
+
+Single core configuration
```
Comment (annwoj) [inline samples/caf_sensor_manager/README.rst:40]:
```suggestion
Single-core configuration
```

File: samples/caf_sensor_manager/README.rst [hunk 2]
```diff
@@ -31,6 +32,24 @@ This module handles the data received from the :ref:`caf_sensor_data_aggregator`
...
+=========================
+
+For the MCU with multiple cores, the default configuration will use one core to simulate the sensor and the other core to process the sensor.
```
Comment (annwoj) [inline samples/caf_sensor_manager/README.rst:43]:
```suggestion
For the MCU with multiple cores, the default configuration uses one core to simulate the sensor and the other core to process the sensor.
```

File: samples/caf_sensor_manager/README.rst [hunk 3]
```diff
@@ -31,6 +32,24 @@ This module handles the data received from the :ref:`caf_sensor_data_aggregator`
...
+These multiple core MCU can support single core configuration, where the sensor is simulated and processed on the single, selected core.
```
Comment (annwoj) [inline samples/caf_sensor_manager/README.rst:44]:
```suggestion
These multicore MCUs can support a single-core configuration, where the sensor is simulated and processed on a single, selected core.
```

File: samples/caf_sensor_manager/README.rst [hunk 4]
```diff
@@ -31,6 +32,24 @@ This module handles the data received from the :ref:`caf_sensor_data_aggregator`
...
+The configuration is placed in the :file:`boards/<board>_singlecore.conf file`.
```
Comment (annwoj) [inline samples/caf_sensor_manager/README.rst:45]:
```suggestion
The configuration is placed in the :file:`boards/<board>_singlecore.conf` file.
```
not sure if the 'file' after .conf was supposed to be kept as a part of the path

**Review by emob-nordic — APPROVED** [review]

**Review by annwoj — APPROVED** [review]

**Pull request merged**